src/dev.docker-compose.yml.jinja

```diff
@@ -95,6 +95,8 @@ services:
       - ./data/addons:/data/odoo/addons
       - ./data/filestore:/data/odoo/filestore
       - ./data/sessions:/data/odoo/sessions
+      # pytest
+      - ./odoo/.pytest_cache:/odoo/.pytest_cache
```

Comment:
I think it's better to create a volume instead of a mount bind.

Reply:
What is the difference ?

Reply:
Mount bind = directory or file (?!) exists in the project directory, it's visible to the user in the file system / editor.
It should exists before runing docker compose and have the right file permissions.

Instead volume is created / managed / prunned automatically by docker.

Reply:
`You can use volumes to define multiple types of mounts; volume, bind, tmpfs, or npipe.`
https://docs.docker.com/reference/compose-file/services/#volumes:~:text=You%20can%20use%20volume,bind

<img width="928" height="205" alt="image" src="https://github.com/user-attachments/assets/712fc318-411b-4a8b-84c5-0685801cae14" />